specutils/io/default_loaders/wcs_fits.py

```diff
@@ -85,9 +90,80 @@ def wcs1d_fits_loader(file_obj, spectral_axis_unit=None, flux_unit=None,
     if not isinstance(file_obj, fits.hdu.hdulist.HDUList):
         hdulist.close()
 
+        if wcs.naxis > 4:
+            raise ValueError('FITS file input to wcs1d_fits_loader is > 4D')
+        elif wcs.naxis > 1:
+            for i in range(wcs.naxis - 1, 0, -1):
+                try:
+                    wcs = wcs.dropaxis(i)
+                except(_wcs.NonseparableSubimageCoordinateSystemError) as e:
+                    raise ValueError(f'WCS cannot be reduced to 1D: {e} {wcs}')
+
     return Spectrum1D(flux=data, wcs=wcs, meta=meta)
 
 
+@custom_writer("wcs1d-fits")
+def wcs1d_fits_writer(spectrum, file_name, update_header=False, **kwargs):
```

Comment:
I think we might want to rename this to something like `image-fits`/`image_fits_writer` (analogous to tabular) -- `wcs1d` should be implicit it because it's on a `Spectrum1D` writer, and I think we might want to express explicitly that it's writing to an images vs a table.

Reply:
Technically there are two characteristic properties – storing the flux spectrum as a 1D image, and encoding the spectral axis as `wcs.WCS`. But I agree the first one is probably more informative to the user, and this might make the documentation on the different loader types clearer or easier to write.

Reply:
So once I have updated both identifiers it would be probably be a good moment to go ahead with the renaming. Are you more in favour of `FITS IMAGE`, `FITS BINTABLE`, `[FITS] IRAF`; or rather `IMAGE-FITS` etc. @nmearl?